src/myself.erl

```diff
@@ -1,3 +1,4 @@
+%% COMMON ELVISCI COMPLAIN ABOUT THIS VERY VERY EXTREMELY INSANELY LONG LONG LINE THAT I JUST WROTE FOR YOU, OK?
```

Comment:
Line 1 is too long: %% COMMON ELVISCI COMPLAIN ABOUT THIS VERY VERY EXTREMELY INSANELY LONG LONG LINE THAT I JUST WROTE FOR YOU, OK?.